diff -Nru domoweb-0.1.99.orig/src/examples/init/domoweb domoweb-0.1.99/src/examples/init/domoweb
--- domoweb-0.1.99.orig/src/examples/init/domoweb	2012-03-05 13:32:32.000000000 +0100
+++ domoweb-0.1.99/src/examples/init/domoweb	2012-03-05 13:35:38.000000000 +0100
@@ -41,7 +41,7 @@
 
 function start_django {
     echo -n " * Starting DomoWeb..."
-    /bin/su - ${DOMOWEB_USER} -c "dmg_domoweb >/var/log/domogik/domoweb.log 2>&1 &"
+    /bin/su - ${DOMOWEB_USER} -c "dmg_domoweb >/var/log/domoweb/domoweb.log 2>&1 &"
     if [ $? -gt 0 ]; then
         echo " An error occured during DomoWeb startup."
         exit 1
